### DISCUSSION TITLE
How long does it take for a merged new version of a manifest to show up?

### DISCUSSION THREAD

**Discussion by rkeithhill-keysight** [microsoft/winget-pkgs#66216] [General] [answered]
My PR to add a new version of our app was merged about 6 hours ago.  I'm still seeing only the previous version.  How long does it typically take for an app version to show up after a PR merge?

Update:
I checked again at 7.5 hours after the merge and the new version has shown up.  I wonder if there is something (pkg index) that gets updated at a certain time of the day e.g. noon PDT.

**Comment by Trenly** [comment, marked as answer]
Package updates generally show up within an hour of being merged. However, there have been issues with the [Publishing Pipeline](https://dev.azure.com/ms/winget-pkgs/_build?definitionId=338&_a=summary) lately which have prevented updates from being pushed out.
